### DISCUSSION TITLE
Need for VSEARCH

### DISCUSSION THREAD

**Discussion by marchoeppner** [bio-raum/FooDMe2#13] [General]
I have been wondering if we could perhaps ditch VSEARCH. It adds some complexity to the processing logic and, as far as I can tell, wouldn't work with Nanopore data. Seems like supporting DADA2 would be enough. What do you think @gregdenay ?

**Comment by gregdenay** [comment]
In FooDMe1 I kept VSEARCH around as legacy support.

The methods were developed with dereplication has standard clustering method and we showed than dada2 and vsearch have the same accuracy (for the 16S meat system at least). I would perhaps make the default processing with dada instead of vsearch because it does improve teh performance a bit in most cases. We might still want to try VSearch on new barcode system in the future so it would be nice to have it around.

**Reply by marchoeppner** [reply]
OK fair enough, then I will switch the logic to make dada the default and instead add a `--vsearch` flag.

**Reply by marchoeppner** [reply]
@gregdenay I switched dada to be the default, so to run vsearch you now need `--vsearch`
